packages/itwin/tree-widget/src/tree-widget-react/components/trees/imodel-content-tree/IModelContentTree.tsx

```diff
@@ -38,7 +38,9 @@ export function IModelContentTree(props: IModelContentTreeProps) {
       treeName={IModelContentTreeComponent.id}
       getHierarchyDefinition={getDefinitionsProvider}
       selectionMode={props.selectionMode ?? "extended"}
-      treeRenderer={(treeProps) => <TreeRenderer {...treeProps} getDecorations={(node) => <Icon href={getIcon(node)} />} />}
+      treeRenderer={(treeProps) => (
+        <TreeRenderer {...treeProps} actions={props.actions} getDecorations={props.getDecorations ?? ((node) => <IModelContentTreeIcon node={node} />)} />
```

Comment:
I guess you should not pass `actions` to `Tree` component if they are used by `TreeRenderer`

Reply:
adjusted